tests/dp_computations_test.py

```diff
@@ -657,5 +657,102 @@ def test_compute_dp_sum_noise_std_gaussian(self, eps: float, delta: float,
         self.assertAlmostEqual(scale, expected_std)
 
 
+class AdditiveMechanismTests(parameterized.TestCase):
+
+    @parameterized.parameters(
+        dict(epsilon=2, l1_sensitivity=4.5, expected_noise=2.25),
+        dict(epsilon=0.1, l1_sensitivity=0.55, expected_noise=5.5),
+    )
+    def test_laplace_mechanism_creation(self, epsilon, l1_sensitivity,
+                                        expected_noise):
+        mechanism = dp_computations.LaplaceMechanism(
+            epsilon=epsilon, l1_sensitivity=l1_sensitivity)
+        self.assertEqual(mechanism.noise_kind, pipeline_dp.NoiseKind.LAPLACE)
```

Comment:
Very nit: please could you add a blank line between "arrange" "act" and  "assert" sections here and in the other tests? I agree that in most of the tests it's hard to mix them up since the code is nice and simple, but just to follow the standard code style conventions.

Reply:
Good point. Done